pkg/blocker/blocker_test.go

```diff
@@ -24,7 +24,7 @@ var (
 	checkTime = time.Millisecond * 100
 	blockTime = time.Second
 	addr      = test.RandomAddress()
-	logger    = log.NewLogger("test", log.WithSink(ioutil.Discard))
+	logger    = log.NewLogger("test", log.WithSink(io.Discard))
```

Comment:
This is a forgotten line from logger refactoring. Please use `log.Noop` instead of `log.NewLogger("test", log.WithSink(io.Discard))`, in a separate PR.